sql_generators/serp_events/templates/metadata.yaml

```diff
@@ -11,6 +11,12 @@ labels:
 scheduling:
   dag_name: bqetl_serp
   task_name: {{ app_name }}_serp_events__v1
+  date_partition_parameter: null
+  destination_table:
+    >-
+    {{ app_name }}_serp_events_v1${{
+    (execution_date - macros.timedelta(hours=24)).strftime("%Y%m%d")
+    }}
```

Comment:
```suggestion
    }}
  query_file_path:
    # explicit query file path is necessary because the destination table
    # includes a partition identifier that is not in the path
    sql/moz-fx-data-shared-prod/{{ app_name }}_derived/serp_events_v1/query.sql
```